game/scripts/vscripts/TutorialGraph/Steps.ts

```diff
@@ -0,0 +1,52 @@
+import { tutStep } from "./Core"
+
+const isHeroNearby = (location: Vector, radius: number) => FindUnitsInRadius(
+    DOTATeam_t.DOTA_TEAM_GOODGUYS, location, undefined, radius,
+    DOTA_UNIT_TARGET_TEAM.DOTA_UNIT_TARGET_TEAM_BOTH,
+    DOTA_UNIT_TARGET_TYPE.DOTA_UNIT_TARGET_BASIC | DOTA_UNIT_TARGET_TYPE.DOTA_UNIT_TARGET_HERO,
+    DOTA_UNIT_TARGET_FLAGS.DOTA_UNIT_TARGET_FLAG_NONE,
+    0, false
+).length > 0
+
+/**
+ * Creates a tutorial step that waits for a hero to go to a location.
+ * @param location Target location
+ */
+export const tutGoToLocation = (location: Vector) => {
+    return tutStep((context, complete) => {
+        Tutorial.CreateLocationTask(location)
+
+        // Wait until a hero is at the goal location
+        const checkIsAtGoal = () => {
+            if (isHeroNearby(location, 200)) {
```

Comment:
@RobinKa I would move this magic 200 to be a optional parameter into the function